docs/dsc_getting_started.md

```diff
@@ -1,4 +1,5 @@
 # What setup is needed to run a configuration?
+- As is this document is written for using DSC standalone. If using these resources in another product such as Ansible or Puppet consult the vendor's documentation.
```

Comment:
```suggestion
- This document is written for using DSC alone. If using these resources in another product such as Ansible or Puppet consult the vendor's documentation.
```

Reply:
agreed and fixed

Reply:
oh I didnt see I could commit straight from your suggestion neat. I pushed the update from my laptop.